modules/improvedigitalBidAdapter.js

```diff
@@ -229,31 +270,40 @@ const ID_REQUEST = {
   buildServerRequests(requestObject, bidRequests, bidderRequest) {
     const requests = [];
     if (config.getConfig('improvedigital.singleRequest') === true) {
-      requestObject.imp = bidRequests.map((bidRequest) => this.buildImp(bidRequest));
-      requests[0] = this.formatRequest(requestObject, bidderRequest);
+      const pbsConfig = this.pbsConfig();
```

Comment:
pbsConfig method is redundant. It's not an object, we just need a simple pbs boolean and pass this boolean as param to other methods. Why would we need an object here after the publisherId was removed?

Reply:
<img width="652" alt="image" src="https://user-images.githubusercontent.com/93644923/162893822-aef9a937-ef06-4c66-bb5b-c30e8275c7df.png">

Overriding the global PBS mode from ad-unit param. Let me know your feedback.